UI/FleetButton.cpp

```diff
@@ -172,13 +172,29 @@ FleetButton::FleetButton(const std::vector<int>& fleet_IDs, SizeType size_type)
 
     // select icon(s) for fleet(s)
     int num_ships = 0;
+    m_fleet_blockaded = false;
     for (std::shared_ptr<const Fleet> fleet : fleets) {
-        if (fleet)
+        if (fleet) {
             num_ships += fleet->NumShips();
+            if (!m_fleet_blockaded && fleet->Blockaded())
+                m_fleet_blockaded = true;
+        }
+    }  
+
+    // add graphics for all icons needed
+    if (m_fleet_blockaded) {
+        std::shared_ptr<GG::Texture> blockaded_texture = FleetBlockadedIcon(size_type);
+        if (blockaded_texture) {
+            GG::StaticGraphic* icon = new GG::StaticGraphic(blockaded_texture, GG::GRAPHIC_FITGRAPHIC | GG::GRAPHIC_PROPSCALE);
+            GG::Clr opposite_clr(255 - this->Color().r, 255 - this->Color().g, 255 - this->Color().b, this->Color().a);
```

Comment:
there isn't some function to find a reverse colour being used for starlane wasted PP indication that could be used here as well? (haven't checked... am just wondering)